assets/components/directDebit/directDebitActions.js

```diff
@@ -12,6 +12,8 @@ export type SortCodeIndex = 0 | 1 | 2;
 export type Action =
   | { type: 'DIRECT_DEBIT_POP_UP_OPEN' }
   | { type: 'DIRECT_DEBIT_POP_UP_CLOSE' }
+  | { type: 'DIRECT_DEBIT_GUARANTEE_OPEN' }
+  | { type: 'DIRECT_DEBIT_GUARANTEE_CLOSE' }
```

Comment:
Since you are adding two actions, can you add the tests for this actions as well? (actions test file: https://github.com/guardian/support-frontend/blob/master/assets/components/directDebit/__tests__/directDebitActionsTest.js)

Reply:
Additionally, it would be nice to update the reducers unit tests as well (reducer test file: https://github.com/guardian/support-frontend/blob/master/assets/components/directDebit/__tests__/directDebitReducerTest.js)

Reply:
Will do

Reply:
Thanks 😄